### PR TITLE
CVE-2018-11784	

### PR DESCRIPTION
**Pull request by iamnukee**

### CODE REVIEW

File: cves/CVE-2018-11784.yml [hunk 1]
```diff
@@ -44,15 +44,19 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description:
```
Comment (Bailey-Pearson) [inline cves/CVE-2018-11784.yml:47]: Could you maybe put this in your own words? This is super similar to the website's description so I don't fully understand it.

File: cves/CVE-2018-11784.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CVE: CVE-2018-11784
+CVE: 2018-11784
```
Comment (Bailey-Pearson) [inline cves/CVE-2018-11784.yml]:
```suggestion
CVE: CVE-2018-11784
```

File: cves/CVE-2018-11784.yml [hunk 3]
```diff
@@ -175,7 +186,7 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
+    applies: true
     note:
```
Comment (Bailey-Pearson) [inline cves/CVE-2018-11784.yml]: You need to add a note to explain how this applies

File: cves/CVE-2018-11784.yml [hunk 4]
```diff
@@ -120,10 +126,12 @@ discovered:
...
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: |
+    Looking at the emails and the initial dates, Jun 5th 2018 seems to be the 
+    closest I'll get to the day this bug was actually discovered. 
+  date: 2018-06-05
```
Comment (Bailey-Pearson) [inline cves/CVE-2018-11784.yml:138]: This date is a lot sooner than the date you said that the bug was reported to the team

File: cves/CVE-2018-11784.yml [hunk 5]
```diff
@@ -120,10 +126,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: |
+    Looking at the emails and the initial dates, Jun 5th 2018 seems to be the
```
Comment (Bailey-Pearson) [inline cves/CVE-2018-11784.yml]:
```suggestion
    Looking at the emails and the initial dates, June 5th 2018 seems to be the 
```

File: cves/CVE-2018-11784.yml [hunk 6]
```diff
@@ -206,4 +217,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: Overall this was a coding mistake and specifically an err in assumptions.
```
Comment (Bailey-Pearson) [inline cves/CVE-2018-11784.yml:228]: Do you have any opinions on their mitigation?

File: cves/CVE-2018-11784.yml (hunk 4 above)
Comment (iamnukee) [reply cves/CVE-2018-11784.yml:138]: This was an err on my part, the reported date should've read the same as the discovered date. It was the earliest sate I found

File: cves/CVE-2018-11784.yml (hunk 6 above)
Comment (iamnukee) [reply cves/CVE-2018-11784.yml:228]: Not really, it wasn't apparent that they tried multiple things to fix it before it worked out. The bug occurred due to lack of carefulness on the path of the developer. They probably weren't thinking about that bug being a possibility.

**Pull request merged**